openedx/core/djangoapps/api_admin/models.py

```diff
@@ -0,0 +1,61 @@
+import logging
+
+from django.contrib.auth.models import User
+from django.db import models
+from django.utils.translation import ugettext as _
+from simple_history.models import HistoricalRecords
+
+
+log = logging.getLogger(__name__)
+
+
+class ApiAccessRequest(models.Model):
+    """Model to track API access for a user."""
+
+    APPROVED = 'approved'
+    DENIED = 'denied'
+    INACTIVE = 'inactive'
+    STATUS_CHOICES = tuple(
+        (status, status.capitalize()) for status in (APPROVED, DENIED, INACTIVE)
+    )
+    user = models.ForeignKey(User)
+    status = models.CharField(
+        max_length=255,
+        choices=STATUS_CHOICES,
+        default=INACTIVE,
+        db_index=True,
+        help_text=_(
+            'Status of this API access request. "Approved" users can access the API, "denied" cannot, and "inactive" '
```

Comment:
It might make more sense to put these descriptions in the display values, rather than the help text.